app/helpers/application_helper.rb

```diff
@@ -174,4 +174,8 @@ def stop_clickable(module_name, topic_name, status)
       end
     end
   end
+
+  def track_analytics?
+    cookies[:track_analytics] == 'Yes'
```

Comment:
We also discussed the option of dropping a case-sensitive string in favour of a boolean value

Reply:
Meaning the `SettingsController` needs to understand "Yes" or "No" but after that we get it coerced to `true` or `false`

Reply:
Inspecting this, it looks like a `true` value gets parsed to a `"true"` string, so i think making it a boolean adds complexity. Do you think i should make any changes to this (e.g. strings `"true"` and `"false"`, or lower case `"yes"` and `"no"`) or leave as is?

Reply:
My preference is to uncouple frontend as much as possible, for example, if we eventually move the "Yes" and "No" strings to locales we are prevented from adding new locales here. We have made our code dependent upon case-sensitive English. Would've been nice if ActiveDispatch or whatever coerced cookie values when read, but at least we can take a step in that direction and not mix things by storing 'true' and 'false'.

Reply:
I've made the change